### ISSUE TITLE
Include license files

### ISSUE THREAD

**Issue by tillkruss** [cachewerk/linux-repos#3]
Can we add a license file to the `.deb` and/or `.rpm` package?

**Comment by michael-grunder** [comment]
Yes, there are mechanisms to do this and actually, we'll need to include a few from the other projects we're using if/when we package them statically.

For `relay-pkg.so` as it is now I think we need to do this for:

* [PHP](https://www.php.net/license/3_01.txt)
* [jemalloc](https://github.com/jemalloc/jemalloc/blob/dev/COPYING)
* [hiredis](https://github.com/redis/hiredis/blob/master/COPYING)
* [ck](https://github.com/concurrencykit/ck/blob/master/LICENSE)

**Comment by tillkruss** [comment]
Resolved in #41 (`edc90d4`).

Packages now ship `/usr/share/doc/<pkg>/copyright` with the Relay EULA, plus the notices for anything statically linked.

Scope was determined from the binaries rather than assumed, and is narrower than the original list here:

| Library | Linkage | Notice shipped |
|---|---|---|
| hiredis | static (`redisConnect` in symtab) | yes |
| ck | static (`ck_rhs_init`) | yes |
| lz4 / zstd | `dlopen("liblz4.so.1")` at runtime | no |
| jemalloc | **absent from all three binaries** | no |
| PHP | linked at runtime by the host process | no |

jemalloc appears nowhere in `relay.so`, `relay-pkg.so` or `relay-zts.so`, so the 2021 comment above is stale on that point. lz4 and zstd became `dlopen`'d in v0.40.0 ("Removed `LZ4` and `ZSTD` dependency and detect them on startup"), so they are the system's own shared libraries.

The notices are conditional on `$pkg_binary`: only `relay-pkg.so` builds bundle anything. The plain `relay.so` used on noble, plucky and trixie links hiredis and ck dynamically and is covered by the distribution's own packages.

One wrinkle worth recording: `ck`'s license embeds an Apache-2.0 notice for `src/ck_hp.c`, which needs `/usr/share/common-licenses/Apache-2.0` referenced or lintian raises `copyright-not-using-common-license-for-apache2`. Vendored files under `build/licenses/` are pristine upstream copies; the Debian pointer is emitted from `helpers.sh`.
